gradle.properties

```diff
@@ -18,6 +18,7 @@
 org.gradle.caching=true
 org.gradle.parallel=true
 org.gradle.configureondemand=true
+org.gradle.jvmargs=-XX:ThreadStackSize=10240
```

Comment:
nit: Should this be `-Xss` since `-XX:flags` are JVM implementation dependent.

Reply:
quote from https://www.oracle.com/technetwork/java/javase/tech/vmoptions-jsp-140102.html:

```
Options that are specified with -XX are not stable and are subject to change without notice
```

Reply:
Ah, OK I will change it. I chose the long form just to be super clear, but stability is more important.